feeds/feed.go

```diff
@@ -12,3 +15,17 @@ type Package struct {
 	CreatedDate time.Time `json:"created_date"`
 	Type        string    `json:"type"`
 }
+
+func NewPackage(created, cutoff time.Time, name, version, feed string) *Package {
+	if created.Before(cutoff) {
+		return nil
+	}
+	log.Println("Processing:", name, version, feed)
```

Comment:
Consider using `log.WithFields(...)` here as seen in `cmd/scheduled-feed/main.go`.
```
log.WithFields(log.Fields{
	"name":         pkg.Name,
	"feed":         pkg.Type,
	"created_date": pkg.CreatedDate,
}).Print("sending package upstream")
```

Reply:
Sure, also with that in mind it might also be a consideration to globally switch to `logrus` throughout